libvast/src/system/exporter.cpp

```diff
@@ -130,8 +130,11 @@ void request_more_hits(stateful_actor<exporter_state>* self) {
 
 caf::settings exporter_state::status() {
   caf::settings result;
+  put(result, "hits-count", rank(hits));
   put(result, "hits", to_string(hits));
```

Comment:
I should have mentioned this in the first review: calling `to_string` on an `ids` instance is problematic, because the output can get *really* large.

How about removing the current `hits` and then just rename `hits-count` to `hits`?